### PR TITLE
Change the client name

### PR DESCRIPTION
**Pull request by devigned**
AuthorizationClient doesn't seem to make sense for what this spec contains.

### CODE REVIEW

**Comment by vivsriaus** [comment]
Thanks @devigned Can you please rename the file as well?

File: arm-resources/authorization/2015-01-01/swagger/authorization.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "AuthorizationClient",
+    "title": "LockManagementClient",
```
Comment (vivsriaus) [inline arm-resources/authorization/2015-01-01/swagger/authorization.json]: I would rename this to ManagementLockClient. Agree, @ravbhatnagar @yangl900 ?

Comment (ravbhatnagar) [inline arm-resources/authorization/2015-01-01/swagger/authorization.json]: Makes sense. Thanks for catching this @devigned.

Comment (devigned) [inline arm-resources/authorization/2015-01-01/swagger/authorization.json]: Thank you for the prompt confirmation. 

I'm going to merge the PR.

**Pull request merged**